pkg/apiserver/apiserver.go

```diff
@@ -507,17 +515,19 @@ func (s *APIServer) waitForResourceSync(ctx context.Context) error {
 		{Group: "network.kubesphere.io", Version: "v1alpha1"}: {
 			"ippools",
 		},
-		{Group: "notification.kubesphere.io", Version: "v2beta1"}: {
+	}
+	if s.Config.MultiClusterOptions.Enable {
```

Comment:
Notification is controlled by `Config.MultiClusterOptions.Enable`, right?

Reply:
I refer to the controller's code:
https://github.com/kubesphere/kubesphere/blob/master/cmd/controller-manager/app/controllers.go#L535

Reply:
It's not a good choice,  the API needs a separate control flag, and the notification controller has a different logic.

Reply:
I redefined a variable that controls whether notifications are turned on or not.
<img width="467" alt="image" src="https://user-images.githubusercontent.com/35139861/196124192-6be41690-b552-41c0-8476-0044dd48c8ad.png">